charts/grafana/templates/grafana-auth-sidecar-configmap.yaml

```diff
@@ -19,8 +19,12 @@ data:
     }
     server {
       server_name {{ template "grafana.url" . }} ;
-      listen 8080  ;
-      listen [::]:8080  ;
+      listen {{ .Values.global.authSidecar.port }}  ;
+      listen [::]:{{ .Values.global.authSidecar.port }}  ;
```

Comment:
Do we need or want ipv6 support? AFAIK Nothing else in our stack has ipv6.

Reply:
removed ipv6 configs from auth-sidecar